src/components/Products.js

```diff
@@ -0,0 +1,181 @@
+            let selItem = data.filter((item)=>{
+                if(item.id)
+                return id == item.id;
+            })
+            if(selItem.length){
+                return selItem[0].id;
+            }
+        } else {
+            return 'default';
+        }
+    }
+    render(){
+        const {homeApi,productInfo,router} = this.props;
+        const {categoryData,productList,openModal} = this.state;
+        return(<div className="products-page">
+            <div className="overlay" style={{display: (productInfo.searching_productdetails|| productInfo.adding_to_cart  || homeApi.categorydata_searching) ? "block" : "none"}}>
+                <div className="loading"><SvgLoading /></div>
+            </div>
+            <Header router={router} cartClick={this.showCartView} cartInfo={productInfo.cartItems}></Header>
+            <div className="row content-area">
+                <aside className="col span-3-of-12 left-menu">
+                    <div className="prod-categories">
+                        <ul className="cat-list">
+                            {categoryData && categoryData.length ? categoryData.filter((item) => { return item.order >= 0 }).map((data, i) => (<li onClick={(event) => this.makeActiveCategory(event, data.id)} className={this.state.catId ? data.id == this.state.catId ? 'selected' : '' : ''} key={i}>
+                                <p>{data.name}</p>
+                            </li>)) : null}
+                        </ul>  
+                        
+                    </div>
+                </aside>
+                <section className='dropdown-cat-list'>
+                    <div className="select">
+                        <select onChange={(event) => { this.makeActiveCategory(event) }} value={this.getDropDwnSelectedVal(this.state.catId, categoryData)} name="slct" id="slct">
+                            <option value="default" disabled>Choose an option</option>
+                            {categoryData && categoryData.length ? categoryData.filter((item) => { return item.order >= 0 }).map((data, i) => (<option value={data.id} className={this.state.catId ? data.id == this.state.catId ? 'selected' : '' : ''} key={i}>
+                                {data.name}
+                            </option>)) : null}                            
+                        </select>
+                    </div>
+                </section>
+                <section className="col span-9-of-12 product-list-area">
+                    <div className="prod-list">
+                        <ul className="rest-tiles">
+                        {productList && productList.length ? productList.filter((data)=>{return this.state.catId ? this.state.catId == data.category : data }).map((item,i)=>(<li key={i}>
+                            <p className="prod-name" title={item.name}>{item.name}</p>
+                            <div className="prod-detls">
+                            <div className="prod-img">
```

Comment:
extract product list to separate component

Reply:
done